mesh/mesh.go

```diff
@@ -402,6 +429,22 @@ func (m *Mesh) AddBlock(blk *types.Block) error {
 	return nil
 }
 
+func (m *Mesh) SetZeroBlockLayer(lyr types.LayerID) error {
```

Comment:
Note that you call this function only from the _already part of the network_ happy flow. If you are syncing and one of the layers is empty, you won't get here.